Sources/xcodeproj/Extensions/Decodable+Dictionary.swift

```diff
@@ -8,8 +8,10 @@ extension Decodable {
     /// - Parameter jsonDictionary: json dictionary.
     /// - Throws: throws an error if the initialization fails.
     init(jsonDictionary: [String: Any]) throws {
```

Comment:
What do you think about turning this into a static method that is more explicit about the type of decodable that is returning?

```swift
static func projectDecodable(dictionary: [String: Any]) throws -> Decodable
```